extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +40,124 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  background: white !important;
+  color: black !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  position: relative !important;
+  width: 100% !important;
+  border-radius: 16px 16px 0px 0px !important;
+  font-size: 24px !important;
+  text-align: center !important;
+  height: 44px !important;
+  background: #EDEDED !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-content {
+  position: relative !important;
+  margin: 28px 28px 24px 28px !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-text {
+  position: relative !important;
+  margin-bottom: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-price, .i-amphtml-story-subscriptions-title,
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  overflow: hidden !important;
+  white-space: nowrap;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.i-amphtml-story-subscriptions-price {
+  width: 100% !important;
+  font-size: 24px !important;
+  margin-top: 28px !important;
+  letter-spacing: -0.4px !important;
+  height: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-title {
+  width: 100% !important;
+  font-weight: bold !important;
+  font-size: 20px !important;
+  letter-spacing: -0.25px !important;
+  height: 41px !important;
+}
+
+.i-amphtml-story-subscriptions-subtitle-first, .i-amphtml-story-subscriptions-subtitle-second {
+  width: 100% !important;
+  font-style: normal !important;
+  font-size: 16px !important;
+  letter-spacing: -0.25px !important;
+  height: 22px !important;
+}
+
+.i-amphtml-story-subscriptions-button-container {
+  display: flex !important;
+  justify-content: space-between !important;
+  flex-direction: column !important;
+  margin-bottom: 40px !important;
+  height: 92px !important;
+  font-family: Google Sans, sans-serif !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-button-container, .i-amphtml-story-subscriptions-google-button-container {
+  display: flex !important;
+  justify-content: center !important;
+}
+
+.i-amphtml-story-subscriptions-publisher-button {
+  display: flex !important;
+  justify-content: center !important;
+
+  width: 212px !important;
```

Comment:
What happens if the text within the button is too long, or too short? Should this width be dynamic, e.g. adapt to the text content but have a max-width, after which the text shows ellipsis?

Reply:
Fixed with fixed width and nowrap.

Reply:
Please be sure [ellipses](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow) display if the text is too long.
<img width="150" alt="Screen Shot 2022-04-06 at 1 34 50 PM" src="https://user-images.githubusercontent.com/3860311/162044764-238a8314-32cd-43b0-a678-2a5362fc811e.png">

Reply:
We need to remove a hard-coded width. Otherwise the button will not shrink to fit its content.
We do need a max width.